src/code_translation/orchestrator.py

```diff
@@ -90,6 +95,11 @@ def transform(src: str) -> str:
         # Export new_code
         new_src = new_tree.code
 
+        # Take care of linting with autoflake
+        # TODO: This does not exactly work as expected, as the requirements for 'unused variables' seem a little weird
+        new_src = autoflake.fix_code(
```

Comment:
If it doesn't work, please open a new branch for it. This is just a dead code now :D

Reply:
> If it doesn't work, please open a new branch for it. This is just a dead code now :D

I mean at least it removes unused imports :sweat_smile:. Yeah but I can also move it to another branch...